mitiq/tests/test_about.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (C) Unitary Fund
+#
+# This source code is licensed under the GPL license (v3) found in the
+# LICENSE file in the root directory of this source tree.
+
+"""Tests for mitiq.about()."""
+
+import mitiq
+
+
+def test_stdout():
+    """Tests function prints a str."""
+    mitiq.about()
```

Comment:
I don't think this tests that `mitiq.about()` prints to standard out (at least when run using `pytest mitiq/tests/test_about.py` which, IMO, we should support).

There is an example of capturing standard output in the pytest docs: https://docs.pytest.org/en/6.2.x/capture.html#accessing-captured-output-from-a-test-function

Reply:
Yes, I did see that example before choosing to use `pytest -rP mitiq/tests/test_about.py`

I'll read more on the differences between capturing an stdout vs. testing an stdout.